app/workers/work-failer.ts

```diff
@@ -51,15 +51,15 @@ export default class WorkFailer implements Worker {
     const workItems = await getWorkItemsByUpdateAgeAndStatus(
       db, lastUpdateOlderThanMinutes, [WorkItemStatus.RUNNING],
       [JobStatus.RUNNING, JobStatus.RUNNING_WITH_ERRORS],
-      ['w.id', 'w.jobID', 'serviceID', 'startedAt'],
+      ['w.id', 'w.jobID', 'serviceID', 'startedAt', 'workflowStepIndex'],
```

Comment:
I'm curious why you needed the workflow step index - in case the same service is used multiple times in the chain? I think the change makes sense for that case, but wondering if there was another reason.

Reply:
There's a function within `getWorkItemsByUpdateAgeAndStatus` that I'm calling in order to get work item counts that requires it.